examples/k8s-service/README.md

```diff
@@ -0,0 +1,275 @@
+# Running the Cloud SQL Proxy as a Service
+
+This example demonstrates how to run the Cloud SQL Auth Proxy with PgBouncer on
+Kubernetes as a service. It assumes you have already successfully completed all
+the steps in [Using the Cloud SQL Auth Proxy on Kubernetes][sidecar].
+
+In this example, you will deploy [PgBouncer][] with the Cloud SQL Auth Proxy as
+a sidecar, in addition to configuring encryption between the application and
+PgBouncer.
+
+## A Word of Warning
+
+Running PgBouncer with the Cloud SQL Auth Proxy may pose a significant
+operational burden and should be undertaken with caution given the attendant
+complexity.
+
+In general, we recommend [running the proxy as a sidecar][sidecar] to your
+application because it is simple, there is less overhead, it is secure out of
+the box, and there is less latency involved.
+
+However, the service pattern is useful when you are at very large scale, when
+you clearly need a database connection pooler, and when you are running into SQL
+Admin API quota problems.
+
+## Initial Setup
+
+Before we deploy PgBouncer with the Cloud SQL Auth Proxy, there are three
+initial steps to take.
+
+### Generate Certificates for PgBouncer
+
+First, you will need to generate certificates to encrypt the connection between
+the application and PgBouncer. We recommend using [CFSSL][] to handle
+certificate generation. Note: this example uses self-signed certificates. In
```

Comment:
Do we recommend it or is that just what this example is using?

Reply:
I'll verify what best-practices are and update the wording to make it clear.